libs/application/core/src/constants/sharedDataProviders.ts

```diff
@@ -0,0 +1,30 @@
+import { Application } from '../types/Application'
+import { DataProviderTemplateApi } from '../types/StateMachine'
+import { PerformActionResult } from '../types/TemplateApiModuleTypes'
+
+export const SharedDataProviders = {
+  nationalRegistryProvider: {
+    apiModuleAction: 'nationalRegistry',
+    namespace: 'nationalRegistry',
+    externalDataId: 'nationalRegistry',
+    shouldPersistToExternalData: true,
+    throwOnError: true,
+    useMockData: false,
+    mockData: (application: Application): PerformActionResult => {
+      return {
+        response: {
+          names: {
+            rolling: 'stones',
+          },
+        },
+        success: true,
+      }
+    },
```

Comment:
Well there you could extract answers from the application and (eg. on the first screen which only renders on dev or sth) or even some environment variable and based on those determine different responses.

Reply:
Sure, but I wonder if that's to manual so only the most ambitious applications will set that up. It would be nice if we could solve 90% of test cases without much boilerplate.